varcode/variant_collection.py

```diff
@@ -92,58 +94,55 @@ def _clone_metadata(self, new_variants):
             variants=new_variants,
             original_filename=self.original_filename)
 
+    @memoize
+    def high_impact_effects(self, *args, **kwargs):
```

Comment:
I think the `*args` and `**kwargs` forwarding is confusing, and I'd expect this method to take a `min_priority` argument that defaults to `Substitution`. Some ideas:

 * Take an explicit `min_priority` and `raise_on_error` arguments.
 * Have a `VariantCollection.raise_on_annotation_error` boolean that user can set to indicate the whether to raise annotation errors on any method call for that object.
 * Have a global `varcode.RAISE_ON_ANNOTATION_ERROR` boolean that anything can look at. If we care about thread saftey (not saying we should), could have a `varcode.set_raise_on_annotation_error(value)` and `varcode.get_raise_on_annotation_error()` function that saves this value in thread local storage.